episodes/severity-static.Rmd

```diff
@@ -0,0 +1,640 @@
+
+One epidemiological approach to estimating the clinical severity is quantifying the Case Fatality Ratio or CFR. CFR is the ratio of deaths from an infectious disease to the number of confirmed diagnosed cases. However, calculating this directly from the cumulative number of deaths to cases during an epidemic tends to result in a biased CFR given the time delay from onset to death, varying substantially as the epidemic progresses and stabilising at the later stages of the outbreak ([Ghani et al., 2005](https://academic.oup.com/aje/article/162/5/479/82647?login=false#620743)).
+
+![Observed biased confirmed case fatality ratio (CFR) estimates as a function of time (thick line) calculated as the ratio of the cumulative number of confirmed cases to deaths at time t. The estimate at the end of an outbreak (i.e. 302/1755 = 17.2 %) is the realised CFR by the end of the epidemic. The horizontal continuous line and dotted lines show the expected value and the 95% confidence intervals (95% CI) of the predicted delay-adjusted CFR estimate (based on Nishiura et al., 2009) only by using the observed data until 27 Mar 2003 (estimated at 18.1 % (95% CI: 10.5, 28.1).](fig/cfr-pone.0006852.g003-fig_c.png)
+
+In this tutorial we are going to learn how to use the `{cfr}` package to calculate and adjust a CFR estimation using [delay distributions](../learners/reference.md#delaydist) from `{epiparameter}` or elsewhere, based on the methods developed by [Nishiura et al., 2009](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0006852), also, how we can reuse `{cfr}` functions for more severity measurements.
+
+We’ll use the pipe `%>%` to connect some of their functions, so let’s also call to the `{tidyverse}` package:
+
+```{r,message=FALSE,warning=FALSE}
+library(cfr)
+library(epiparameter)
+library(tidyverse)
+```
+
+## Naive CFR
+
+We measure disease severity in terms of case fatality ratio (CFR) --which is given by the ratio of the cumulative number of deaths $D_{t}$ to the cumulative number of confirmed cases $C_{t}$ at a certain time $t$. We can refer to the _naive CFR_ (also crude or biased CFR, $b_{t}$):
+
+$$ b_{t} =  \frac{D_{t}}{C_{t}} $$
+
+<!-- total number of deaths $D$ divided by total number of confirmed cases $C$ -->
+
+:::::::::::::: callout
+
+### What are data sources to estimate CFR?
+
+[Verity et al., 2020](https://www.thelancet.com/journals/laninf/article/PIIS1473-3099(20)30243-7/fulltext) summaries the spectrum of COVID-19 cases.
```

Comment:
I would suggest moving it to an earlier concepts focused section, as I'm not sure it is adding something specific here.

Reply:
I also found this useful. I moved as a section before the naive CFR section in https://github.com/epiverse-trace/tutorials-middle/pull/1/commits/f368cfd7e86d8b9c674973ff38c52a2dfd625c42